.eslintrc.json

```diff
@@ -45,7 +45,7 @@
     "react/react-in-jsx-scope": "off",
 
     "jsx-a11y/click-events-have-key-events": "warn",
-    "jsx-a11y/no-autofocus": "warn",
+    "jsx-a11y/no-autofocus": "off",
```

Comment:
disabling `jsx-a11y/no-autofocus` is acceptable. In some scenarios, setting autofocus is necessary for better UX. For example, forms in modals or dialogs can focus on the first input when they appear.